### PR TITLE
[JA] Bump up basics/mix-tasks to 1.0.2

### PR DESCRIPTION
**Pull request by Tsuyoshi84**
Translated the mix-tasks page to follow the latest version 1.0.2.
Please review the pull request @elixirschool/japanese

This pull request includes the following changes.
- https://github.com/elixirschool/elixirschool/commit/633a3ebe5fb8495b31937fa7e3fa4198cf4bdf9b#diff-ce81ef91d95296ee6fc601cb0c462f42
- https://github.com/elixirschool/elixirschool/commit/575bf3bf036a8dd23687426c0b53a536aba8dd6d#diff-ce81ef91d95296ee6fc601cb0c462f42
- https://github.com/elixirschool/elixirschool/commit/8a8f948636d464505a47856cd05728504fe30d78#diff-ce81ef91d95296ee6fc601cb0c462f42

Thank you 😊

### CODE REVIEW

**Comment by riseshia** [comment]
I think some changes not included in this pr. i.e. https://github.com/elixirschool/elixirschool/commit/633a3ebe5fb8495b31937fa7e3fa4198cf4bdf9b#diff-ce81ef91d95296ee6fc601cb0c462f42R64

could you check this? XD

**Comment by Tsuyoshi84** [comment]
Oh, seems like somehow I reverted one change that I should have made. 

I added the missing change at line 64. 
Thank you for your careful checking!

File: ja/lessons/basics/mix-tasks.md [hunk 1]
```diff
@@ -111,3 +121,6 @@ mix app.start         # Starts all registered apps
 mix hello             # Simply calls the Hello.say/0 function.
 ...
 ```
+
+注: 新しいタスクが `mix help` の出力に表示されるには、私たちのコードが事前にコンパイルされてなければいけません。
```
Comment (riseshia) [inline ja/lessons/basics/mix-tasks.md]: In my opinion,`私たちの` is not not necessary in Japanese, How do you think?

File: ja/lessons/basics/mix-tasks.md [hunk 2]
```diff
@@ -111,3 +121,6 @@ mix app.start         # Starts all registered apps
...
+私たちは `mix compile` を直接実行するか、先ほどと同じように `mix hello` を実行してコンパイルのトリガーを作り出すことでコンパイルすることができます。
```
Comment (riseshia) [inline ja/lessons/basics/mix-tasks.md]: Same here `私たちは`

Comment (riseshia) [reply ja/lessons/basics/mix-tasks.md]: (maybe) `コンパイルのトリガーを作り出すことでコンパイルすることができます。` -> 
`コンパイルできます。`
It seems too verbose.

**Comment by Tsuyoshi84** [comment]
I modified the translation.
Please check it out!

**Pull request merged**